features/octavia/gunicorn/config.pan

```diff
@@ -0,0 +1,22 @@
+unique template features/octavia/gunicorn/config;
+
+
+# Include gunicorn base configuration
+include 'features/openstack/gunicorn/config';
+
+
+# Define gunicorn configuration for Octavia
+include 'components/metaconfig/config';
+prefix '/software/components/metaconfig/services/{/etc/sysconfig/gunicorn.octavia}';
+'module' = 'tiny';
+# panlint disable=LP006
+bind '/software/components/metaconfig/services/{/etc/sysconfig/gunicorn.octavia}/contents' = openstack_gunicorn_sysconfig;
+# Doesn't work as it is not unescaped by metaconfig. Uncomment after 24.10 release.
+#'daemons' = {
+#    SELF[escape('gunicorn@octavia')] = 'restart';
+#    SELF;
+#}; 
```

Comment:
As this should be fixed in 24.10, can this be uncommented now?

Reply:
I hesitated but I'd prefer to test it first. It is not really critical and I think it may even not be needed as `gunicorn` tends to reload services affected by a change...

Reply:
Okay, I'd rather avoid shipping commented out code.